src/components/header/header.jsx

```diff
@@ -88,27 +109,39 @@ Header.NavLinks = ({
   handleMenu,
   openSignUpModal,
   openLoginModal,
+  openMoviePreviewModal,
 }) => {
+  const { playSound } = useSounds();
   const navLinks = [
     {
       title: "Slop Search",
       src: headerMagnifyglass,
       link: "/search",
+      onClick: () => {},
```

Comment:
Does adding an `onClick` to one link mean we _have_ to define it for all the others, even if we don't need them? Ideally we don't add code that we don't need/that has no purpose.

Reply:
No, it's not necessary to have an empty onClick function for the other links. I've updated the code to remove them!